src/lib/utils/functions/getDataFromFromStorage.ts

```diff
@@ -0,0 +1,24 @@
+import { EntityHeader, UserProfile } from '../synapseTypes'
+import { SynapseConstants } from '..'
+
+export const getStoredEntityHeaders = (): EntityHeader[] => {
+    try {
+      const lookUpEntityHeaders: EntityHeader[] = JSON.parse(
+        sessionStorage.getItem(SynapseConstants.ENTITY_HEADER_STORAGE_KEY) || '',
```

Comment:
Can change at some point from `SynapseConstants.ENTITY_HEADER_STORAGE_KEY) || ''` to `SynapseConstants.ENTITY_HEADER_STORAGE_KEY) ?? ''`